CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ It diverges in the following ways:
 ## Unreleased
 ### Changed
 - (disabled by default) Amazon Device Messaging (ADM) is supported in release builds (#2781)
+- (disabled by default) FFTV is now opened automatically if a tab is sent while the app is backgrounded
```

Comment:
Since we'll be enabled in this release, we can remove the "disabled by default" comment.

@dnarcese You should remove these (disabled by default) as part of relman, cleaning up the changelog!